ResourcesCard.jsx

```diff
@@ -1 +1,83 @@
+import React, { useEffect } from "react";
+
+import { sendEvt } from "services/event.service";
+
+import imgFallback from "img/resource_fallback.png";
+
+import styles from "./ResourcesCard.module.scss";
+
+const ResourcesCard = () => {
+  const [resources, setResources] = useState([]);
+  const [primaryResource, setPrimaryResource] = useState(none);
+  const [secondaryResources, setSecondaryResources] = useState([]);
+
+  var sendEventTracking = (resourceUuid) => {
+    const evt = new Object();
+    evt = {
+      event_name: "web_dashboard_card_" + r.id + "_" + r.title + "_tap"
+    };
+    sendEvt(evt);
+    console.log("sending event");
+  };
+
+  // TODO
+  useEffect(async () => {
+    let req = await getResources(userId);
+    setResources(req.resources);
+
+    let primaryResource = resources[0];
+    let secondaryResources = resources.slice(1, 5);
+    setPrimaryResource(primaryResource);
+    setSecondaryResources(secondaryResources);
+    return;
+  });
+
+  // Get image of first resource
+  const i = () => {
+    const image = primaryResource.icon;
+    if (!primaryResource.icon || primaryResource.icon === undefined) {
+      return imgFallback;
+    }
+
+    if (image === true) {
+      return image;
+    }
```

Comment:
With the destructure change above, this can just be 
```suggestion
    return imgFallback || image
```